src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,35 @@
+import Container from '@mui/material/Container';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+
+import './Footer.scss';
+
+import companyLogoWhite from '../../assets/company-logo-white.png';
+import footerConfig from '../../data/footerConfig.json';
+
+const { logo, companyName } = footerConfig;
+
+
+function Footer() {
+
+  return (
+    <footer className="footer">
+      <Container maxWidth="xl">
+        <Box display="flex" alignItems="center" justifyContent="space-between">
+          <Box display="flex" alignItems="center">
```

Comment:
Usestyled components instead
You can reuse that if required

Reply:
Will replace all the scss with `styled-components` in that case